### DISCUSSION TITLE
Question about potential incompatibility with mpv.net

### DISCUSSION THREAD

**Discussion by Sneakpeakcss** [tomasklaen/uosc#615] [Q&A]
From what i've gathered uosc is developed with mpv in mind, so rather than opening an unwanted Issue i suppose this is a better place to ask.
 

While using uosc with mpv.net i'm unable to properly navigate uosc menus/ use volume-bar by mouse:

https://github.com/tomasklaen/uosc/assets/77424331/805e2962-56bf-4ebe-b5e4-9c5055a583cf

As seen on video it looks like for some reason mouse_up action isn't properly detected, and creates this buggy behaviour where the whole part is just locked in place following mouse position, the moment an additional key(can even be unbinded) is pressed it releases the unregistered mouse_up. Surprisingly going back to a previous part of the menu works perfectly with a single mouse click.

I assume that this might have something to do with uosc drag detection(?) since this only happens for everything that's on the 'drag-zone' only when in window mode, and only with mouse clicks, additionally the key-press-workaround seems to stop working after i move mouse cursor by around 20px, however this is only true for uosc-menu, pressing any keyboard key releases volume-bar no matter what.

I couldn't manage to make it work with any provided settings, clean installs without any additional plugins also doesn't help.
So my question is… Is there any obvious reason for this? Would there be any potentially easy edit i could do to get this working for myself?

**Comment by tomasklaen** [comment]
I don't have much input on what might be wrong as I don't use mpv.net, but I'm surprised you're having this problem, because I thought there were more people using uosc+mpv.net combo, and this is a uosc breaking bug that would definitely come up before. Are you sure this is not something with mpv.net settings? Maybe there's an issue with solutions about it on their repo.

To me, it sounds like mpv.net is preventing mouse up events in some circumstances.

**Comment by Sneakpeakcss** [comment]
> I don't have much input on what might be wrong as I don't use mpv.net, but I'm surprised you're having this problem, because I thought there were more people using uosc+mpv.net combo, and this is a uosc breaking bug that would definitely come up before. Are you sure this is not something with mpv.net settings? Maybe there's an issue with solutions about it on their repo.
> 
> To me, it sounds like mpv.net is preventing mouse up events in some circumstances.

I've tried my luck with changing config options, but didn't find any exclusive ones in mpv.net that would have anything to do with input, and even with a clean installation (default settings with no additions) it behaves the same, so yeah… I had the same idea thinking that if it happens with default config then someone would notice it, but the last mentions of uosc in mpv.net's issue tracker are mostly redirections here, and the last real reported issue about uosc goes years back.

Though i've managed to find this https://github.com/mpvnet-player/mpv.net/issues/533 which seems like something that might be connected, but once again, while there's not that many spots on internet where people discuss mpv/mpv.net, it's kind of weird that there's not even a single mention of it since uosc is usually the single first UX script that's recommended.

Maybe someone else here uses it, and could confirm if they can reproduce this on their side.
